modules/libffi/3.4.7.bcr.4/MODULE.bazel

```diff
@@ -0,0 +1,44 @@
+module(
+    name = "libffi",
+    version = "3.4.7.bcr.4",
+    bazel_compatibility = [">=7.2.1"],
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "platforms", version = "0.0.11")
+bazel_dep(name = "rules_cc", version = "0.1.1")
+bazel_dep(name = "rules_license", version = "1.0.0")
+
+http_archive = use_repo_rule("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "libffi-x86-64bit-msvc-binaries",
+    build_file_content = """
+cc_import(
+    name = "libffi",
+    hdrs = ["ffi.h","ffitarget.h"],
+    interface_library = "libffi-8.lib",
+    shared_library = "libffi-8.dll",
+    visibility = ["//visibility:public"],
+)
+""",
+    sha256 = "3f218abf32c765a6229ecc04d83d67e90266025e3f79b752358771119d02d8fb",
+    strip_prefix = "",
+    urls = ["https://github.com/libffi/libffi/releases/download/v3.5.2/libffi-3.5.2-x86-64bit-msvc-binaries.zip"],
```

Comment:
This version doesn't match the package version. Please either use the corresponding version or update the package version.